packages/legend-query-builder/src/components/fetch-structure/QueryBuilderTDSWindowPanel.tsx

```diff
@@ -89,14 +95,20 @@ const createWindowColumnState = (
       operator,
       columnState,
     );
+    const columnName = operator.isColumnAggregator()
```

Comment:
maybe we could aggregate line 98 and 109 together to create columnName above if, but current code looks good too 
just a personal preference